source/pymongo.txt

```diff
@@ -42,27 +42,31 @@ You must install the PyMongo driver module to make it available to your
 Python application. We recommend using
 `pip <http://pypi.python.org/pypi/pip>`__ to install PyMongo.
 
+PyMongo requires `dnspython <https://pypi.org/project/dnspython>`__  to support 
+``mongodb+srv://`` connection strings and `MongoDB Atlas 
+<https://www.mongodb.com/cloud>`__.
```

Comment:
Question: Does making it a "required dependency" as stated in the ticket mean they need to explicitly install it or does it get installed when you run pip?
If the former, would it be possible to show an example of how to install it for readers that might not know as well as those that want the copy-paste convenience?
If the latter, could that be stated here?